utils/test_utils.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/forbole/juno/v2/cmd/parse"
+	dbconfig "github.com/forbole/juno/v2/database/config"
+	"github.com/forbole/juno/v2/logging"
+
+	junodb "github.com/forbole/juno/v2/database"
+
+	"github.com/forbole/bdjuno/v2/database"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	"github.com/stretchr/testify/suite"
+)
+
+func NewTestDb(suite *suite.Suite, schema string) (*database.Db, codec.Codec) {
```

Comment:
Why would function for creating test db instance return also codec.Codec ? Just use simapp.MakeTestEncodingConfig() where you need it.